src/main/webapp/jquery.quickflip/jquery.quickflip.js

```diff
@@ -0,0 +1,328 @@
+                            // handle any looping of the animation
+                            switch( repeater ) {
+                                case 0:
+                                case -1:
+                                    $.quickFlip.flip( i, null, -1);
+                                    break;
+                                
+                                //stop if is last flip, and attach events for msie
+                                case 1: 
+                                    break;
+                                    
+                                default:
+                                    $.quickFlip.flip( i, null, repeater - 1);
+                                    break;
+                            }
+                        }
+                    });
+                }
+            });
+            
+        },
+        
+        // attach click handlers
+        attachHandlers : function($the_cta, i, panel) {
+            //attach flip
+            $the_cta.click(function(ev) {
+                ev.preventDefault();
+                $.quickFlip.flip(i);
+            });
+        },
+        
+        removeFlipDivs : function(i) {
+            for ( var j = 0; j < $.quickFlip.wrappers[i].flipDivs.length; j++ ) $.quickFlip.wrappers[i].flipDivs[j].remove();
+        },
+        
+        compareObjs : function(obj1, obj2) {
+            if (!obj1 || !obj2 || !obj1.length || !obj2.length || obj1.length != obj2.length) return false;
+            
+            for ( var i=0; i < obj1.length; i++ ) {
+                if (obj1[i]!==obj2[i]) return false;
+            }
+            return true;
+        },
+        
+        combineOptions : function( opts1, opts2 ) {
+            opts1 = opts1 || {};
+            opts2 = opts2 || {};
+            
+            for ( x in opts1 ) {
+                opts2[x] = opts1[x];
+            }
+            
+            return opts2;
+        }
+    };
+
+    $.fn.quickFlip = function( options ) {
+        this.each( function() {
+            new $.quickFlip.init( options, this );
+        });
+        
+        return this;
+    };
+    
+    $.fn.whichQuickFlip = function() {
+        var out = null;
+        
+        for ( var i=0; i < $.quickFlip.wrappers.length; i++ ) {
+            if ( $.quickFlip.compareObjs(this, $( $.quickFlip.wrappers[i].wrapper)) ) out = i;
+        }
+        
+        return out;
+    };
+    
+    $.fn.quickFlipper = function( options, nextPanel, repeater ) {
+        this.each( function() {
+            var $this = $(this);
+            var thisIndex = $this.whichQuickFlip();
+            
+            // if doesnt exist, set it up
+            if ( thisIndex == null ) {
+                $this.quickFlip( options );
+                
+                thisIndex = $this.whichQuickFlip();
+            }
+            
+            $.quickFlip.flip( thisIndex, nextPanel, repeater, options );
+        });
+    };
+    
+})( jQuery );
```

Comment:
Add a newline here, at the end of the file.

Reply:
Done :)